Base/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:jammy-20231004
+FROM ubuntu:jammy-20231128
 LABEL authors="Selenium <selenium-developers@googlegroups.com>"
 
 # Arguments to define the version of dependencies to download
 ARG VERSION
 ARG RELEASE=selenium-${VERSION}
 ARG OPENTELEMETRY_VERSION=1.31.0
-ARG GRPC_VERSION=1.59.0
+ARG GRPC_VERSION=1.60.0
```

Comment:
I am not sure why this was in 1.59.0, but this should be aligned with the upstream repo https://github.com/SeleniumHQ/selenium/blob/selenium-4.16.1-java/java/maven_deps.bzl#L32

Reply:
Yes, let me take note of this one and revert the same version upstream in the next deploy.